CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,185 @@
+# JSON Schema Organizational Code of Conduct
+
+Our (The JSON Schema Organization) Code of Conduct is a combination of
+the "Contributor Covenant Code of Conduct" 2.1 and the IETF BCP 54 "IETF Guidelines for Conduct" (RFC7154).
+
+# IETF Guidelines for Conduct
+
+Note: These guidelines are not provided in full in this document.
+Some extracts are provided.
+Please see the [RFC document](https://www.rfc-editor.org/rfc/rfc7154.html) for full details.
+While we do not currently formally interact with the IETF beyond publication of specification documents,
+we have found their approach to developing specifications well informed and useful.
+
+This document provides a set of guidelines for personal interaction
```

Comment:
The first sentence is a bit awkward reading to me, given "this document" is clearly an extract -- maybe worth eliding just a bit?

Reply:
Yeah I hear you there.
I'll modify slightly.